core/sys_context.go

```diff
@@ -64,6 +64,32 @@ func (sc *SysContractCallCtx) IsWhitelisted(feeCurrency *common.Address) bool {
 
 // GetGasPriceMinimum retrieves gas price minimum for given fee currency address.
 func (sc *SysContractCallCtx) GetGasPriceMinimum(feeCurrency *common.Address) *big.Int {
+	return sc.gasPriceMinimums.GetGasPriceMinimum(feeCurrency)
+}
+
+type GasPriceMinimums map[common.Address]*big.Int
```

Comment:
i'd move this  a few lines below, after GetCurrentGasPriceMinimumMap, so type defition and its method are all together

Reply:
done